Aquality.Selenium.Core/src/Aquality.Selenium.Core/Utilities/JsonSettingsFileExtensions.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using Aquality.Selenium.Core.Configurations;
+
+namespace Aquality.Selenium.Core.Utilities
+{
+    public static class JsonSettingsFileExtensions
```

Comment:
actually this is a SettingsFileExtensions, not necessarily Json

Reply:
fixed